opal/mca/common/cuda/common_cuda.c

```diff
@@ -300,6 +303,15 @@ void mca_common_cuda_register_mca_variables(void)
                                  MCA_BASE_VAR_SCOPE_READONLY,
                                  &mca_common_cuda_cumemcpy_timing);
 #endif /* OPAL_ENABLE_DEBUG */
+
+#if OPAL_CUDA_GPU_MEM_WAR
+    (void) mca_base_var_register("ompi", "mpi", "common_cuda", "gpu_mem_check_war",
+                                 "Set to 1 to enable GPU memory check war",
+                                 MCA_BASE_VAR_TYPE_INT, NULL, 0, 0,
+                                 OPAL_INFO_LVL_9,
```

Comment:
I would like to leave at 9.  I am basically adding this MCA parameter to give myself an easy way to turn off this workaround. There are performance issues so if folks run into them, I can say, turn of this code and see if that fixes it.  Then I can say, update to CUDA 7.5 and everything will be great again.